spec/redux/actionCreators-test.js

```diff
@@ -6,137 +6,104 @@ import thunk from 'redux-thunk';
 import Immutable from 'immutable';
 
 describe("The actionCreators'", () => {
-
   const middlewares = [thunk];
   const mockStore = configureStore(middlewares);
+  var store;
+
+  beforeEach(() => {
+    store = mockStore({});
+  });
+
+  describe("configureIDE function", () => {
+    var resolve, reject;
 
-  describe("configureIDE", () => {
-    var store;
     beforeEach(() => {
-      var runtimeApiLoader = () => new Promise(function(resolve, reject) {});
-      store = mockStore({});
+      var runtimeApiLoader = () => new Promise(function(_resolve, _reject) {
+        resolve = _resolve;
+        reject = _reject;
+      });
       store.dispatch(actCreators.configureIDE({
-        codemirrorOptions: {mode: "javascript"},
+        codemirrorOptions: {mode: 'javascript'},
         runtimeApiLoader: runtimeApiLoader
       }));
     });
 
-    it("dispatches a START_LOAD_RUNTIME action first", () => {
-      expect(store.getActions()[0]).toEqual({type: "START_LOAD_RUNTIME"});
-    });
+    describe("asynchronously dispatches the loadRuntimeApi function", () => {
```

Comment:
moved the loadRuntimeApi tests to inside the configureIDE describe block, since loadRuntimeApi is called within configureIDE. This also allows us to define runtimeApiLoader once.